muikku-core-plugins/src/main/java/fi/otavanopisto/muikku/plugins/profile/ProfileBackingBean.java

```diff
@@ -99,7 +112,19 @@ public String init() {
     
     SchoolDataIdentifier identifier = new SchoolDataIdentifier(userEntity.getDefaultIdentifier(), userEntity.getDefaultSchoolDataSource().getIdentifier());
     emails = userEmailEntityController.getUserEmailAddresses(identifier);
+   
+    chatEnabled = false;
+    String enabledUsersCsv = pluginSettingsController.getPluginSetting("chat", "enabledUsers");
+    
+    if (enabledUsersCsv == null) {
```

Comment:
Might as well check with StringUtils.isEmpty(enabledUsersCsv) to deal with both a null or an empty string

Reply:
Also to avoid future confusion, move lines 122-126 inside the if i.e.
`if (StringUtils.isNotBlank(enabledUsersCsv) {
  // Lines 122-126
}`
and no `return` clause inside there (breaks the execution of the backing bean init).